sim/Makefile

```diff
@@ -0,0 +1,7 @@
+all: client server
+
+server: server.c
+	gcc -g -o server server.c
+
+client: client.c
+	gcc -g -o client client.c
```

Comment:
MAJOR NIT clean is always appreaciated

.PHONY: clean
clean:
  rm server client

Reply:
This is generally not compiled at all.  I wanted to test, and I knew it would take ages for me to get a proper IMS test setup, so I wrote a simulator.  I only included it here because it was too potentially useful to just delete.